cSpell.json

```
@@ -0,0 +1,159 @@
+{
+	"version": "0.2",
+	"language": "en",
+	"words": [
+		"Avalonia",
+		"binlog",
+		"Blazor",
+		"blockquotes",
+		"Bootstrapper",
+		"brotli",
+		"browsersync",
+		"Callout",
+		"chatops",
+		"codebases",
+		"Codespace",
+		"Codespaces",
+		"Contoso",
+		"Cupertino",
+		"customizability",
+		"databinding",
+		"datacontext",
+		"datagrid",
+		"devs",
+		"Dismissable",
+		"Docfx",
+		"ellipsize",
+		"Expando",
+		"Framebuffer",
+		"Geolocation",
+		"globbing",
+		"Gyrometer",
+		"Hanja",
+		"hectopascals",
+		"laggy",
+		"layouter",
+		"layouting",
+		"Listview",
+		"LLRM",
+		"mergeable",
+		"MSAL",
+		"msbuild",
+		"MVVM",
+		"NETSDK",
+		"netstandard",
+		"oidc",
+		"Omnisharp",
+		"overscroll",
+		"Packt",
+		"parameterless",
+		"pickable",
+		"Pluralsight",
+		"POSIX",
+		"reentrantly",
+		"roadmap",
+		"sandboxed",
+		"Segoe",
+		"Silverlight",
+		"Skia",
+		"skiasharp",
+		"Storyboarded",
+		"struct",
+		"Syncfusion",
+		"templatable",
+		"templating",
+		"timespan",
+		"Tizen",
+		"TLDR",
+		"triaging",
+		"UI's",
+		"unmanaged",
+		"Uno's",
+		"unoplatform",
+		"UWP's",
+		"VSIX",
+		"WEBASSEMBLY",
+		"walkthrough"
+	],
+	"ignoreWords": [
+		"ADAL",
+		"Authenticode",
+		"automagically",
+		"Divio's",
+		"emcc",
+		"emscripten",
+		"Flatpickr",
+		"Giesel",
+		"GPIO",
+		"hdpi",
+		"Headered",
+		"Hoefling",
+		"HRESULT",
+		"IDBFS",
+		"Infragistics",
+		"jonathanpeppers's",
+		"Krueger",
+		"libgtk",
+		"Liu's",
+		"Matteo",
+		"McCaffrey",
+		"mdpi",
+		"Mergify",
+		"netstd",
+		"nventive",
+		"slnf",
+		"Stetho",
+		"Talkin",
+		"UADO",
+		"Udemy",
+		"winui"
+	],
+	"patterns": [
+		{
+			"name": "Markdown links",
+			"pattern": "\\((.*)\\)",
+			"description": ""
+		},
+		{
+			"name": "Markdown code blocks",
+			"pattern": "/^(\\s*`{3,}).*[\\s\\S]*?^\\1/gmx",
+			"description": "Taken from the cSpell example at https://cspell.org/configuration/patterns/#verbose-regular-expressions"
+		},
+		{
+			"name": "Inline code blocks",
+			"pattern": "\\`([^\\`].*?)\\`",
```

The failure in CI is because this regex isn't accurate.
This suggestion makes the regex more accurate.

*Note: If you'll commit from GitHub, don't forget to follow conventional commits.*

```suggestion
			"pattern": "\\`([^\\`\\r\\n]+?)\\`",
```